### PR TITLE
Reclaimer update

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Allow reclaimer to convert logs.
Increase storage to 1000 from 100
Remove beakers and buckets from recycle

## Why / Balance
Improvements

## How to test
Use the reclaimer

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
N/A

**Changelog**
:cl:
- tweak: Reclaimers can now process logs and empty beakers, jugs, and buckets.
- tweak: Reclaimers liquid storage increased to 1000u.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Machines/material_reclaimer.yml [hunk 1]
```diff
@@ -66,6 +66,7 @@
       components:
       - PhysicalComposition
       - SpaceGarbage
+      - Logs # Frontier
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Structures/Machines/material_reclaimer.yml:69]: Should be Log.
```suggestion
      - Log # Frontier
```

File: Resources/Prototypes/Entities/Structures/Machines/material_reclaimer.yml [hunk 2]
```diff
@@ -77,6 +78,8 @@
       - Brain
       tags:
       - HighRiskItem
+      - Bucket # Frontier
+      - GlassBeaker # Frontier
```
Comment (whatston3) [inline Resources/Prototypes/Entities/Structures/Machines/material_reclaimer.yml:82]: Might look into why this processes syringes when empty but not full - could be another approach for this.  Ideally, I think unused, empty containers could be recycled, particularly jugs.

**Review by whatston3 — CHANGES_REQUESTED** [review]
Works well when Logs changed to Log.  Looking into syringes, the behaviour with them seems desirable.

**Review by whatston3 — COMMENTED** [review]
Bigger reservoir's great, log processing's fine.

I don't understand why empty containers (beakers, buckets, jugs) shouldn't be processed.  Empty containers, _especially jugs_, should be considered as garbage and easily disposed.  Full ones, sure, leave 'em be, make sure you can't use the reclaimer as a cheap bluespace beaker, but you should still be able to dispose of the things.

Why not both remove PhysicalComposition and add TrashOnEmptyComponent onto containers?

Also, is the largest issue with the beaker the potential for garbage chem, or is it the constant spilling?  Should the reclaimer be spillable onto a drain?  That way, you can maintain it, tip it over occasionally, or harvest the fluids (artisanal trash cocktails with #1475).  Gives the janitor something to do at the very least.

**Comment by whatston3** [comment]
I've put up a set of changes on my repo at [whatston3:2024-06-12-Reclaimer-Empty-Containers](https://github.com/whatston3/frontier-station-14/tree/2024-06-12-Reclaimer-Empty-Containers).  I'd like you to test it - behaviourally, I think it makes sense - in terms of maintainability, not sure if it meets your standards.

One side effect of this would be that trigger happy janitors (or magnetic trash bags) can cause havoc in a laboratory, but that seems like a small price to pay to eliminate the mountain of empty jugs you get as a chemist.

**Comment by dvir001** [comment]
> Bigger reservoir's great, log processing's fine.
> 
> I don't understand why empty containers (beakers, buckets, jugs) shouldn't be processed. Empty containers, _especially jugs_, should be considered as garbage and easily disposed. Full ones, sure, leave 'em be, make sure you can't use the reclaimer as a cheap bluespace beaker, but you should still be able to dispose of the things.
> 
> Why not both remove PhysicalComposition and add TrashOnEmptyComponent onto containers?
> 
> Also, is the largest issue with the beaker the potential for garbage chem, or is it the constant spilling? Should the reclaimer be spillable onto a drain? That way, you can maintain it, tip it over occasionally, or harvest the fluids (artisanal trash cocktails with #1475). Gives the janitor something to do at the very least.

The bucket and beakers issue is that you can insert them by mistake when you click on the container and destroy them

**Comment by dvir001** [comment]
Testing this

**Comment by whatston3** [comment]
> The bucket and beakers issue is that you can insert them by mistake when you click on the container and destroy them
Given the comment in MaterialReclaimerSystem, I'm calling that a bug.  I've got a fix for that on that branch I mentioned.  Give it a look when you can.

**Comment by dvir001** [comment]
> > The bucket and beakers issue is that you can insert them by mistake when you click on the container and destroy them
> 
> Given the comment in MaterialReclaimerSystem, I'm calling that a bug. I've got a fix for that on that branch I mentioned. Give it a look when you can. (edit: damnit, beat me to it)

Tested and seems fine now

**Review by whatston3 — APPROVED** [review]
Retested, works a treat!  Gonna have a quick look around for other side-effects and merge it if I can't find anything.

**Comment by whatston3** [comment]
Think Goblins might be able to eat syringes on that food rework branch, and the idea of a little green homunculus chomping on beaker fragments is funny to me.

Outside of the janitor's trash bag and that, I think we're good.

**Pull request merged**